index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Some meta tags -->
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portofolio Website</title>
+    <!-- Link to the Font-Awesome Font -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.2/css/font-awesome.min.css" />
+    <!-- Link to Roboto Font-->
+    <link href="http://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css" />
+    <!-- Styles -->
+    <!-- Custom Styles -->
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <nav class="nav-container">
+        <ul class="navbar-list">
+            <li class="name1">John Deo</li>
+            <li class="hamburger-menu"><i class="fa fa-bars"></i></li>
+        </ul>
+    </nav>
+    <header class="header-container">
+        <div class="content">
+            <div class="ellipse">
+                <img src="./images/Ellipse-1.svg" alt="" />
+            </div>
+            <div class="personal-description">
+                <div class="name2">
+                    <h1>Laura Jong</h1>
+                </div>
+                <div class="occupation">
+                    <h2>Software Developer</h2>
+                </div>
+                <div class="description">
+                    <p>
+                        I can help you build a product , feature or website Look through of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </header>
```

Comment:
- [ ] It would be nice if you use the `<section>` tag instead of the `<header>` tag in this section as it's a block of your code related to a certain idea.